core/src/test/scala/unit/kafka/server/ReplicationQuotasTest.scala

```diff
@@ -33,6 +33,7 @@ import org.apache.kafka.clients.producer.{KafkaProducer, ProducerRecord}
 import org.junit.Assert._
 import org.junit.{After, Before, Test}
 import scala.collection.JavaConverters._
+import scala.collection.immutable.Range.Inclusive
```

Comment:
Can you please fix the package name (it should not have `unit.`)? Maybe worth checking if any other ones were introduced like that.